src/components/ha-hls-player.ts

```diff
@@ -186,6 +186,14 @@ class HaHLSPlayer extends LitElement {
     });
   };
 
+  private _isLLHLSSupported(): boolean {
+    // LL-HLS requires use of an http/2 proxy to avoid opening too many connections
+    return (
+      performance.getEntriesByType("navigation")[0] &&
```

Comment:
Add a guard for older browsers to make sure performance is defined
```suggestion
      "performance" in window &&
      performance.getEntriesByType("navigation")[0] &&
```